resources/assets/uad_lists.json

```
@@ -37452,5 +37461,212 @@
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.qualcomm.qti.gpudrivers.taro.api31",
+    "list": "Misc",
+    "description": "Adreno Graphics Drivers\nGPU drivers",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.thundercomm.ar.core",
+    "list": "Oem",
+    "description": "Looks like a debugging app.\nIt's AR (Augmented Reality)?",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.android.theme.font.ArchivoSemiBold",
+    "list": "Oem",
+    "description": "Font ArchivoSemiBold",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.android.fota",
+    "list": "Carrier",
+    "description": "AT&T Software Update, But by the carrier.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.overlay.googleasi",
+    "list": "Oem",
+    "description": "Overlay to Android System Intelligence 'com.google.android.as', not needed.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.motointelligence.overlay",
+    "list": "Oem",
+    "description": "Overlay to 'com.motorola.motointelligence', not needed.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.android.overlay.deviceconfig.manaus",
+    "list": "Oem",
+    "description": "Random configurations to basic things(powersaving, network, camera, etc.).",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Unsafe"
+  },
+  {
+    "id": "com.motorola.android.overlay.lhbm",
+    "list": "Oem",
+    "description": "Not sure if its for high brightness mode, still unknown\n'config_udfps_local_hbm_supported' 'true'",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.motorola.motolights",
+    "list": "Oem",
+    "description": "Moto Lights\nPeople recommend to remove it.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.motorola.android.settings.overlay.eqs",
+    "list": "Oem",
+    "description": "Overlay to refresh rate change in settings.\nNot sure if it's unused.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.motorola.sstservice",
+    "list": "Oem",
+    "description": "Spatial Sound\nProvides sst audio effects.\nI am not sure how to use this app.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
This is a feature for spatial sound. Removing this will break it, so it shouldn't be `Recommended`.

---

um hard to say, code is too small, no activities only some frameworks in classes.dex

---

Just make it `Advanced` to be sure (you can commit my suggestion). If people want to, they can still uninstall/disable it.